### PR TITLE
Fel Openshift update

### PR DESCRIPTION
**Pull request by felderi**
Fel Openshift update

### CODE REVIEW

File: app/enterprise/1.3-x/kong-for-kubernetes/install.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ This installation topic guides you through installing and deploying Kong for Kub
 
 <img src="https://doc-assets.konghq.com/kubernetes/K4K8S-Enterprise-Diagram.png" alt="Kong for Kubernetes Enterprise control diagram">
 
->Note: Installation steps in this topic include installing Kong for Kubernetes Enterprise using YAML. Other deployment options, such as using Helm Chart and Kustomize, will be available at a later time.
+>Note: Installation steps in this topic include installing Kong for Kubernetes Enterprise using YAML using kubectl and for OpenShift oc. Other deployment options, such as using Helm Chart and Kustomize, will be available at a later time.
```
Comment (lena-larionova) [inline app/enterprise/1.3-x/kong-for-kubernetes/install.md:10]:
```suggestion
>Note: Installation steps in this topic include installing Kong for Kubernetes Enterprise using YAML with kubectl and with OpenShift oc. Other deployment options, such as using Helm Chart and Kustomize, will be available at a later time.
```

File: app/enterprise/1.3-x/kong-for-kubernetes/install.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ This installation topic guides you through installing and deploying Kong for Kub
...
 
 
 ### Prerequisites
 Before installing Kong for Kubernetes Enterprise, be sure you have the following:
 
-- **Kubernetes cluster**: You can use [Minikube](https://kubernetes.io/docs/setup/minikube/) or a [GKE](https://cloud.google.com/kubernetes-engine/) cluster. Kong is compatible with all distributions of Kubernetes. 
-- **kubectl access**: You should have `kubectl` installed and configured to communicate to your Kubernetes cluster.
+- **Kubernetes cluster**: Kong is compatible with all distributions of Kubernetes. You can use [Minikube](https://kubernetes.io/docs/setup/minikube/), [GKE](https://cloud.google.com/kubernetes-engine/) or [OpenShift](https://www.openshift.com/products/container-platform) cluster.
```
Comment (lena-larionova) [inline app/enterprise/1.3-x/kong-for-kubernetes/install.md:16]:
```suggestion
- **Kubernetes cluster**: Kong is compatible with all distributions of Kubernetes. You can use a [Minikube](https://kubernetes.io/docs/setup/minikube/), [GKE](https://cloud.google.com/kubernetes-engine/), or [OpenShift](https://www.openshift.com/products/container-platform) cluster. 
```

File: app/enterprise/1.3-x/kong-for-kubernetes/install.md [hunk 3]
```diff
@@ -35,7 +35,11 @@ first provision the `kong` namespace:
 
 ```bash
 $ kubectl create namespace kong
-namespace/kong created
+```
+
+On OpenShift:
+```bash
+oc new-project kong
```
Comment (lena-larionova) [inline app/enterprise/1.3-x/kong-for-kubernetes/install.md:42]:
```suggestion
$ oc new-project kong
```

**Comment by lena-larionova** [comment]
@felderi I went ahead and made some small grammatical/formatting tweaks to get this published. I'll copy it over to the 1.5.x docs as well afterwards.

File: app/enterprise/1.3-x/kong-for-kubernetes/install.md [hunk 4]
```diff
@@ -7,17 +7,17 @@ This installation topic guides you through installing and deploying Kong for Kub
...
+- **kubectl or oc access**: You should have `kubectl` or `oc` (if working with OpenShift) installed and configured to communicate to your Kubernetes cluster.
 - A valid Kong Enterprise License
-  * If you have a license, continue to [Step 1. Set Kong Enterprise License](#step-1-set-kong-enterprise-license) below. If you need your license file information, contact Kong Support. 
-  * If you need a license, request a trial license through our [Request Demo](https://konghq.com/request-demo/) page. 
+  * If you have a license, continue to [Step 1. Set Kong Enterprise License](#step-1-set-kong-enterprise-license) below. If you need your license file information, contact Kong Support.
+  * If you need a license, request a trial license through our [Request Demo](https://konghq.com/request-demo/) page.
   * Or, try out Kong for Kubernetes Enterprise using a live tutorial at [https://kubecon.konglabs.io/](https://kubecon.konglabs.io/)
 - Kong Enterprise Docker registry access
```
Comment (GayleNeumann) [inline app/enterprise/1.3-x/kong-for-kubernetes/install.md:22]:
```suggestion
- Kong Enterprise Docker registry access.
```

**Pull request merged**